recipes/pkg-config/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: pkg-config
+  version: 0.28
+
+source:
+  fn: pkg-config-0.28.zip
+  url: https://github.com/tpn/pkg-config/archive/pkg-config-0.28.zip
+
+build:
+  number: 2
+  detect_binary_files_with_prefix: 1
+
+requirements:
+  build:
+    - zlib
+    - autoconf
+    - python
+    - automake
+    - libtool
+  run:
+    - zlib
+
+test:
+  commands:
+    - pkg-config --help
+    - !!str [ "$(pkg-config freetype2 --cflags)" == "-I$PREFIX/include/freetype2 -I$PREFIX/include" ]
```

Comment:
Looks like the `ruamel.yaml` linter is parsing this incorrectly. It doesn't realize that `!!str` makes the rest of the line of string. There does seem to be a recent bug fix release for it, but don't know if it fixes this issue or not. Is there a reason `pyyaml` is not used here?